diff -Naur fbpanel-7.2.orig/panel/panel.c fbpanel-7.2/panel/panel.c
--- fbpanel-7.2.orig/panel/panel.c	2020-06-12 21:05:30.000000000 +0300
+++ fbpanel-7.2/panel/panel.c	2025-04-04 14:00:17.798683571 +0300
@@ -525,28 +525,6 @@
     RET(FALSE);
 }
 
-static gboolean
-panel_scroll_event(GtkWidget *widget, GdkEventScroll *event, panel *p)
-{
-    int i;
-
-    ENTER;
-    DBG("scroll direction = %d\n", event->direction);
-    i = p->curdesk;
-    if (event->direction == GDK_SCROLL_UP || event->direction == GDK_SCROLL_LEFT) {
-        i--;
-        if (i < 0)
-            i = p->desknum - 1;
-    } else {
-        i++;
-        if (i >= p->desknum)
-            i = 0;
-    }
-    Xclimsg(GDK_ROOT_WINDOW(), a_NET_CURRENT_DESKTOP, i, 0, 0, 0, 0);
-    RET(TRUE);
-}
-
-
 static void
 panel_start_gui(panel *p)
 {
@@ -567,8 +545,6 @@
         (GCallback) panel_configure_event, p);
     g_signal_connect(G_OBJECT(p->topgwin), "button-press-event",
         (GCallback) panel_button_press_event, p);
-    g_signal_connect(G_OBJECT(p->topgwin), "scroll-event",
-        (GCallback) panel_scroll_event, p);
 
     gtk_window_set_resizable(GTK_WINDOW(p->topgwin), FALSE);
     gtk_window_set_wmclass(GTK_WINDOW(p->topgwin), "panel", "fbpanel");
diff -Naur fbpanel-7.2.orig/plugins/deskno/deskno.c fbpanel-7.2/plugins/deskno/deskno.c
--- fbpanel-7.2.orig/plugins/deskno/deskno.c	2020-06-12 21:05:30.000000000 +0300
+++ fbpanel-7.2/plugins/deskno/deskno.c	2025-04-04 14:02:13.029323643 +0300
@@ -40,15 +40,6 @@
     ENTER;
     change_desktop(dc, 1);
 }
- 
-static gboolean
-scrolled(GtkWidget *widget, GdkEventScroll *event, deskno_priv *dc)
-{
-    ENTER;
-    change_desktop(dc, (event->direction == GDK_SCROLL_UP
-            || event->direction == GDK_SCROLL_LEFT) ? -1 : 1);
-    return FALSE;
-}
 
 static gint
 name_update(GtkWidget *widget, deskno_priv *dc)
@@ -81,8 +72,6 @@
     gtk_button_set_relief(GTK_BUTTON(dc->main), GTK_RELIEF_NONE);
     g_signal_connect(G_OBJECT(dc->main), "clicked", G_CALLBACK(clicked),
         (gpointer) dc);
-    g_signal_connect(G_OBJECT(dc->main), "scroll-event", G_CALLBACK(scrolled),
-        (gpointer) dc);
     dc->namew = gtk_label_new("ww");
     gtk_container_add(GTK_CONTAINER(dc->main), dc->namew);
     gtk_container_add(GTK_CONTAINER(p->pwid), dc->main);
diff -Naur fbpanel-7.2.orig/plugins/deskno2/deskno2.c fbpanel-7.2/plugins/deskno2/deskno2.c
--- fbpanel-7.2.orig/plugins/deskno2/deskno2.c	2020-06-12 21:05:30.000000000 +0300
+++ fbpanel-7.2/plugins/deskno2/deskno2.c	2025-04-04 14:03:36.727064293 +0300
@@ -61,23 +61,6 @@
     RET();
 }
 
-
-static gboolean
-scroll (GtkWidget *widget, GdkEventScroll *event, deskno_priv *dc)
-{
-    int dno;
-    
-    ENTER;
-    dno = dc->dno + ((event->direction == GDK_SCROLL_UP) ? (-1) : (+1));
-    if (dno < 0)
-        dno = dc->dnum - 1;
-    else if (dno == dc->dnum)
-        dno = 0;
-    Xclimsg(GDK_ROOT_WINDOW(), a_NET_CURRENT_DESKTOP, dno, 0, 0, 0, 0);
-    RET(TRUE);
-
-}
-
 static int
 deskno_constructor(plugin_instance *p)
 {
@@ -89,7 +72,6 @@
     gtk_container_set_border_width(GTK_CONTAINER(dc->main), 0);
     //gtk_button_set_alignment(GTK_BUTTON(dc->main), 0, 0.5);
     g_signal_connect(G_OBJECT(dc->main), "clicked", G_CALLBACK (clicked), (gpointer) dc);
-    g_signal_connect(G_OBJECT(dc->main), "scroll-event", G_CALLBACK(scroll), (gpointer) dc);
     
     update_all(dc->main, dc);
   
